sktime/datasets/tests/test_single_problem_loaders.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+"""Test single problem loaders with varying return types."""
+import numpy as np
+import pandas as pd
+import pytest
+
+from sktime.datasets import (  # Univariate; Unequal length; Multivariate
+    load_acsf1,
+    load_arrow_head,
+    load_basic_motions,
+    load_italy_power_demand,
+    load_japanese_vowels,
+    load_osuleaf,
+    load_plaid,
+    load_unit_test,
+)
+
+UNIVARIATE_PROBLEMS = [
+    load_acsf1,
+    load_arrow_head,
+    load_italy_power_demand,
+    load_osuleaf,
+    load_unit_test,
+]
+MULTIVARIATE_PROBLEMS = [
+    load_basic_motions,
+]
+UNEQUAL_LENGTH_PROBLEMS = [
+    load_plaid,
+    load_japanese_vowels,
+]
+
```

Comment:
more consistent way: `mark.parametrize` over the `mtype`/`return_type`, and use `check_is_mtype`